src/api/pros.py

```diff
@@ -0,0 +1,99 @@
+from flask import Blueprint, jsonify, request, abort
+import uuid
+import datetime
+import json
+from utils import load_json, save_json
+
+pros = Blueprint('pros', __name__, url_prefix='/pros')
+
+
+@pros.route('/', methods=['GET'])
+def list_pros():
+    """
+    Fetches list of professionals
+    """
+    db = load_json('./src/api/db.json')
+    if not db:
+        abort(500)
+
+    return jsonify(db), 200
+
+
+@pros.route('/', methods=['POST'])
+def create_pro():
+    """
+    Adds a new professional to directory
+    """
+    data = request.get_json()
+    if not data:
+        abort(400)
+
+    uid = uuid.uuid1()
+    if not uid:
```

Comment:
Can this fail? Documentation doesn't seem to define any failure cases. Did you run into an issue?

Reply:
Also, if you want to just the str identifier, may be you can just do `str(uuid.uuid1()` here.

Reply:
Resolved